content/seismic/reflection/seismic_reflection_common_middle_points.rst

```diff
@@ -1,6 +1,6 @@
-.. _seismic_reflection_common_reflection_points:
+.. _seismic_reflection_common_middle_points:
 
-Common Reflection Points
+Common Midpoint Points
```

Comment:
maybe "Common Midpoint Gathers" (midpoint points is a bit repetitive). Thoughts @dougoldenburg?